core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -154,18 +155,26 @@ object TestUtils extends Logging {
     servers.map(s => formatAddress(s.config.hostName, s.boundPort(SecurityProtocol.SSL))).mkString(",")
   }
 
+  def getSaslBrokerListStrFromServers(servers: Seq[KafkaServer]): String = {
+    servers.map(s => formatAddress(s.config.hostName, s.boundPort(SecurityProtocol.PLAINTEXTSASL))).mkString(",")
+  }
+
   /**
    * Create a test config for the given node id
    */
   def createBrokerConfig(nodeId: Int, zkConnect: String,
     enableControlledShutdown: Boolean = true,
     enableDeleteTopic: Boolean = false,
-    port: Int = RandomPort, enableSSL: Boolean = false, sslPort: Int = RandomPort, trustStoreFile: Option[File] = None): Properties = {
+    port: Int = RandomPort, enableSasl:Boolean = false, saslPort:Int = RandomPort, enableSSL: Boolean = false,
+    sslPort: Int = RandomPort, trustStoreFile: Option[File] = None): Properties = {
+
     val props = new Properties
     var listeners: String = "PLAINTEXT://localhost:"+port.toString
     if (nodeId >= 0) props.put("broker.id", nodeId.toString)
     if (enableSSL)
       listeners = listeners + "," + "SSL://localhost:"+sslPort.toString
+    if (enableSasl)
+      listeners = listeners + "," + "PLAINTEXTSASL://localhost:" + saslPort.toString
```

Comment:
I think it would be more readable if `listeners` were a `Seq` and we can build the `String` at the end with `mkString`